web/templates/user_pill.hbs

```diff
@@ -0,0 +1,15 @@
+<a data-user-id="{{user_id}}" class="view_user_profile pill" tabindex="0">
```

Comment:
We do not use this template anywhere else currently except in `user_display_only_pill`. Is this extracted out considering some future use-case.

Reply:
`user_pill.hbs` is removed the main purpose of that file was to use the same code in if and else block of `is_bot` section. However the `is_bot` is removed so this file is also not needed.

Posted a comment below kindly reply.